assets/build/install.sh

```diff
@@ -47,6 +47,14 @@ exec_as_git git config --global core.autocrlf input
 exec_as_git git config --global gc.auto 0
 exec_as_git git config --global repack.writeBitmaps true
 
+# shallow clone gitlab-ce
+echo "Cloning gitlab-ce v.${GITLAB_VERSION}..."
+exec_as_git git clone -q -b v${GITLAB_VERSION} --depth 1 ${GITLAB_CLONE_URL} ${GITLAB_INSTALL_DIR}
+
+GITLAB_SHELL_VERSION=$(cat ${GITLAB_INSTALL_DIR}/GITLAB_SHELL_VERSION)
```

Comment:
We could do this. But first we should check if an VERSION has been set from the Dockerfile.